code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -37,7 +37,7 @@
 	///Are we active?
 	var/excited = FALSE
 	///Our gas mix
-	var/datum/gas_mixture/turf/air
+	var/datum/gas_mixture/air
```

Comment:
what?

Reply:
something was setting it to `immutable` and nothing actually used the default typepath here for instantiation with an implied `new`
think it was uh
```dm
/turf/open/space/Initialize(mapload)
	SHOULD_CALL_PARENT(FALSE)
	air = space_gas
```
notably `/datum/gas_mixture/immutable/space` is not a subtype of `/datum/gas_mixture/turf`